src/pages/RepoPage/CoverageOnboarding/OtherCI/OtherCI.tsx

```diff
@@ -32,7 +32,7 @@ function OtherCI() {
   const uploadToken = orgUploadToken ?? data?.repository?.uploadToken ?? ''
   const tokenCopy = orgUploadToken ? 'global' : 'repository'
 
-  const uploadCommand = `codecovcli upload-process -t ${uploadToken}${
+  const uploadCommand = `./codecov -u https://qa.codecov.dev upload-process -t ${uploadToken}${
```

Comment:
Please make sure to take a look at this.

Reply:
we're sending the user to qa.codecov.dev?

Reply:
At least on the figma there looks to be "hooky" on the screenshot after the token. Is that required still?

Reply:
Yeah, I feel like I need a url variable here instead. What's the "hooky"?

Reply:
I don't know if putting `qa.codecov.dev`, this should probably be `api.codecov.io`, we limit who can upload, and this will be confusing to end users if we tell them to upload to a place where they can't.

cc @spalmurray-codecov